activiti-cloud-full-example/values.yaml

```diff
@@ -95,30 +98,20 @@ application:
     #     value: "*"
 
 infrastructure:
-  activiti-keycloak:
+  alfresco-identity-service:
+    enabled: true
     keycloak:
-      enabled: true
       keycloak:
-        ingress:
-          enabled: true
-          path: /
-          proxyBufferSize: "16k"
-          hosts:
-            - "activiti-keycloak.REPLACEME"
-          annotations:
-            kubernetes.io/ingress.class: nginx
-            nginx.ingress.kubernetes.io/rewrite-target: /
-            nginx.ingress.kubernetes.io/configuration-snippet: |
-              more_set_headers 'Access-Control-Allow-Methods: "POST, GET, OPTIONS, PUT, PATCH, DELETE"';
-              more_set_headers 'Access-Control-Allow-Credentials: true';
-              more_set_headers 'Access-Control-Allow-Headers: "DNT,X-CustomHeader,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Authorization,authorization"';
-              more_set_headers 'Access-Control-Allow-Origin: $http_origin';
-            nginx.ingress.kubernetes.io/proxy-buffer-size: "16k"
-        preStartScript: |
-          /opt/jboss/keycloak/bin/add-user.sh -u admin -p admin
-          /opt/jboss/keycloak/bin/add-user-keycloak.sh -r master -u admin -p admin
-          cp /realm/activiti-realm.json .
-          sed -i 's/placeholder.com/*/g' activiti-realm.json
+        hosts: *keycloakhost
+    realm:
+      alfresco:
+        client:
+          redirectUris:
+            - "http://activiti-keycloak.REPLACEME/*"
```

Comment:
These are also should be using common chart to be injected into Keycloak deployment via realm template.